4-storage/relational-database-shredder/src/main/scala/com.snowplowanalytics.snowplow.storage/spark/ShredJob.scala

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright (c) 2012-2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+package com.snowplowanalytics.snowplow.storage.spark
+
+import java.io.{PrintWriter, StringWriter}
+import java.util.UUID
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.snowplowanalytics.iglu.client.{JsonSchemaPair, ProcessingMessageNel, Resolver, SchemaKey}
+import com.snowplowanalytics.iglu.client.validation.ProcessingMessageMethods._
+import com.snowplowanalytics.snowplow.enrich.common.FatalEtlError
+import com.snowplowanalytics.snowplow.enrich.common.utils.shredder.Shredder
+import com.snowplowanalytics.snowplow.enrich.common.outputs.{BadRow, EnrichedEvent}
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.serializer.KryoSerializer
+import org.apache.spark.sql.{SQLContext, SparkSession}
+import scalaz._
```

Comment:
I don't have a strong opinion for either.
This approach just seems more compact and gets rid of the import clutter faster imho.